README.md

```diff
@@ -119,6 +119,28 @@ var lightbox1 = $('.lighbox-1 a').simpleLightbox();
 var lightbox2 = $('.lighbox-2 a').simpleLightbox();
 ```
 
+### Customization
+
+You can customize Simplelightbox by changing the style in simplelightbox.css.
+
+If you are using SASS, you can customize Simplelightbox with the following variables
+
+```SASS
+$sl-font-family: inherit;

```

Comment:
You are right. I will merge all the changes by hand and test them, before they go live ;)